src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+// This component is used as Homepage
+
+import React from 'react';
+// import Book from './Book';
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase, if this highlighted code is not required for the functioning of your project i kindly suggest you remove it :+1: :sparkles: